csp_collector.go

```diff
@@ -77,6 +77,9 @@ func validateViolation(r CSPReport) error {
 		"chromenull://",
 		"resource://",
 		"about:blank",
+		"android-webview",
+		"ms-browser-extension",
+		"http://adblockers.opera-mini.net",
```

Comment:
Sorry - I just meant the `http://adblockers.opera-mini.net` entry. 

> see https://github.com/nico3333fr/CSP-useful/tree/master/csp-wtf#opera-mini-double-wtf and other issues

I did check out this README and commit but neither really explained what it is and why it would be showing up a a violation. I'll try to reach out to someone at Opera and see what they have to say on the matter before I accept this one.

If you want to remove `http://adblockers.opera-mini.net` entry for now, I'm happy to merge this and once we get a response we can re-assess the situation.